src/apps/material-search/MaterialSearchInputs.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { FC } from "react";
+import { useText } from "../../core/utils/text";
+import { Work } from "../../core/utils/types/entities";
+
+interface MaterialSearchInputsProps {
+  searchInput: string;
+  setSearchInput: (value: string) => void;
+  availableMaterialTypes: string[] | null;
+  selectedMaterialType: string | null;
+  handleUpdateMaterialType: (newValue: string) => void;
+  work: Work | null;
+}
+
+const MaterialSearchInputs: FC<MaterialSearchInputsProps> = ({
+  searchInput,
+  setSearchInput,
+  availableMaterialTypes,
+  selectedMaterialType,
+  handleUpdateMaterialType,
+  work
+}) => {
+  const t = useText();
+  return (
+    <div className="material-search__inputs-container">
+      <label className="material-search__label" htmlFor="material-search-input">
+        {t("materialSearchSearchInputText")}
+        <input
+          id="material-search-input"
```

Comment:
I don't know if it is important for now, but you will later need Cypress data attributes for testing purposes.
`data-cy=""`

Reply:
True. I won't create tests as part of this PR though.